internal/handlers/pg/msg_update.go

```diff
@@ -103,6 +103,17 @@ func (h *Handler) MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		if u, err = common.GetOptionalParam(update, "u", u); err != nil {
 			return nil, err
 		}
+
+		// get comment from options.Update().SetComment() method
+		if sp.Comment, err = common.GetOptionalParam(document, "comment", sp.Comment); err != nil {
+			return nil, err
+		}
+
+		// get comment from query, e.g. db.collection.UpdateOne({"_id":"string", "$comment: "test"},{$set:{"v":"foo""}})
+		if sp.Comment, err = common.GetOptionalParam(q, "$comment", sp.Comment); err != nil {
```

Comment:
If a comment is set for both update document and query, we replace the comment from the document? 
Is it on purpose? Or should we concatenate with a previous comment or something like that?

The current implementation is the same as for `msg_find.go` (the comment is rewritten there as well), so the question goes to @AlekSi.

Reply:
Is this case possible at all, or is `$comment` only for queries? (I can't find an example with `update` in docs).

Reply:
Doesn't that function run only for a single query?